### PR TITLE
Merge from Staging 1

### PR DESCRIPTION
**Pull request by inirafli**
Merge the EmpowerU app from the staging branch.

### CODE REVIEW

File: local.properties
Comment (akbarmaliki10) [inline local.properties:1]: Maybe this shouldn't be pushed?

image

Comment (inirafli) [reply local.properties:1]: are you sure 😮

**Comment by akbarmaliki10** [comment]
LGTM 
image

**Review by akbarmaliki10 — APPROVED** [review]
image

**Pull request merged**